### ISSUE TITLE
Error: All currency tokens in the order must be the same token

### ISSUE THREAD

**Issue by jpbm** [ProjectOpenSea/opensea-js#733]
Getting error above, using the following parameters to call "createBuyOrder":

```
seaport.createBuyOrder({
                asset: {
                    tokenId: 100,
                    tokenAddress: '0x8a90CAb2b38dba80c64b7734e58Ee1dB38B8992e'
                },
                accountAddress: walletProvider.getAddress(),
                startAmount:0.01,
            })
```

(Not the actual tokenID or startAmount). Full error message:

```
DEPRECATED: Please use providerUtils.standardizeOrThrow() instead
DEPRECATED: Please use providerUtils.standardizeOrThrow() instead
Error: All currency tokens in the order must be the same token
    at Seaport.<anonymous> (/Users/jpbm/nft-bot/node_modules/@opensea/seaport-js/lib/seaport.js:282:35)
    at step (/Users/jpbm/nft-bot/node_modules/@opensea/seaport-js/lib/seaport.js:44:23)
    at Object.next (/Users/jpbm/nft-bot/node_modules/@opensea/seaport-js/lib/seaport.js:25:53)
    at fulfilled (/Users/jpbm/nft-bot/node_modules/@opensea/seaport-js/lib/seaport.js:16:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Thank you!

**Comment by ahmetsahin0** [comment]
same error

**Comment by AsadRasheed-AR** [comment]
Same error, Can someone from the OS team provide feedback on this issue?

**Comment by mr-friend** [comment]
Related to #734 
#740 might fix this as well.

**Comment by PierreUlysse** [comment]
having the same error, but only if I use a ubuntu, anyone having a clue ?

**Comment by sunwrobert** [comment]
This will be fixed in #746

**Comment by parkj90** [comment]
I'm still hitting the same error after upgrading to version 4.0.11 which I believe should have the fix from #746. Can anybody else verify if they're still getting the same error or if it's cleared up for them?
